doc/jenkins-guide.md

```diff
@@ -200,6 +201,21 @@ launchctl start org.nodejs.osx.jenkins
 ~iojs/start.sh
 ```
 
+### Read-only filesystem
+
+Sometimes a failure in the filesystem will cause it to enter a read-only state.
+If that happens, follow the steps below:
+
+```bash
+touch foo                      # to confirm system is read-only, don't proceed if this succeeds
+sudo df                        # to determine the device for `/`
+sudo e2fsck -y /dev/mmcblk0p2  # replace mmcblk0p2 with proper device for `/`
+touch foo                      # this should succeed now
+```
+
+If the second touch fails, follow instructions below to 
```

Comment:
I think a reboot after a ro root should be mandatory, it's chaos for state management and will likely leave you in an inconsistent place.

I'd go as far as to say that an ro root is suggestive of bigger problems that might need further investigation; maybe a reprovision of the machine.

Reply:
Updated the instructions to reboot afterwards and to open an issue to investigate what caused the root fs to become read only.